packages/github/src/github.ts

```diff
@@ -1,158 +1,36 @@
-    super(GitHub.getOctokitOptions(GitHub.disambiguate(token, opts)))
-
-    this.graphql = GitHub.getGraphQL(GitHub.disambiguate(token, opts))
-  }
-
-  /**
-   * Disambiguates the constructor overload parameters
-   */
-  private static disambiguate(
-    token: string | Octokit.Options,
-    opts?: Octokit.Options
-  ): [string, Octokit.Options] {
-    return [
-      typeof token === 'string' ? token : '',
-      typeof token === 'object' ? token : opts || {}
-    ]
-  }
-
-  private static getOctokitOptions(
-    args: [string, Octokit.Options]
-  ): Octokit.Options {
-    const token = args[0]
-    const options = {...args[1]} // Shallow clone - don't mutate the object provided by the caller
-
-    // Base URL - GHES or Dotcom
-    options.baseUrl = options.baseUrl || this.getApiBaseUrl()
-
-    // Auth
-    const auth = GitHub.getAuthString(token, options)
-    if (auth) {
-      options.auth = auth
-    }
-
-    // Proxy
-    const agent = GitHub.getProxyAgent(options.baseUrl, options)
-    if (agent) {
-      // Shallow clone - don't mutate the object provided by the caller
-      options.request = options.request ? {...options.request} : {}
-
-      // Set the agent
-      options.request.agent = agent
-    }
-
-    return options
+const baseUrl = Utils.getApiBaseUrl()
+const defaults = {
+  baseUrl,
+  request: {
+    agent: Utils.getProxyAgent(baseUrl)
   }
+}
 
-  private static getGraphQL(args: [string, Octokit.Options]): GraphQL {
-    const defaults: GraphQLRequestParameters = {}
-    defaults.baseUrl = this.getGraphQLBaseUrl()
-    const token = args[0]
-    const options = args[1]
-
-    // Authorization
-    const auth = this.getAuthString(token, options)
-    if (auth) {
-      defaults.headers = {
-        authorization: auth
-      }
-    }
-
-    // Proxy
-    const agent = GitHub.getProxyAgent(defaults.baseUrl, options)
-    if (agent) {
-      defaults.request = {agent}
-    }
-
-    return graphql.defaults(defaults)
-  }
-
-  private static getAuthString(
-    token: string,
-    options: Octokit.Options
-  ): string | undefined {
-    // Validate args
-    if (!token && !options.auth) {
-      throw new Error('Parameter token or opts.auth is required')
-    } else if (token && options.auth) {
-      throw new Error(
-        'Parameters token and opts.auth may not both be specified'
-      )
-    }
-
-    return typeof options.auth === 'string' ? options.auth : `token ${token}`
-  }
-
-  private static getProxyAgent(
-    destinationUrl: string,
-    options: Octokit.Options
-  ): http.Agent | undefined {
-    if (!options.request?.agent) {
-      if (httpClient.getProxyUrl(destinationUrl)) {
-        const hc = new httpClient.HttpClient()
-        return hc.getAgent(destinationUrl)
-      }
-    }
-
-    return undefined
-  }
-
-  private static getApiBaseUrl(): string {
-    return process.env['GITHUB_API_URL'] || 'https://api.github.com'
-  }
-
-  private static getGraphQLBaseUrl(): string {
```

Comment:
This was removed, octokit takes care of it: https://github.com/octokit/core.js/issues/65